engine/src/main/java/org/terasology/rendering/dag/FinalPostProcessingNode.java

```diff
@@ -143,14 +145,14 @@ private void renderFinalMonoImage() {
     // TODO: have a flag to invert the eyes (Cross Eye 3D), as mentioned in
     // TODO: http://forum.terasology.org/threads/happy-coding.1018/#post-11264
     private void renderFinalStereoImage(RenderingStage renderingStage) {
-        if (postProcessor.isNotTakingScreenshot()) {
-            sceneFinal.bind();
-        } else {
-            ocUndistorted.bind();
-        }
-
         switch (renderingStage) {
             case LEFT_EYE:
+                if (postProcessor.isNotTakingScreenshot()) { // TODO: verify if this works
```

Comment:
Since I can not verify _(Oculus VR Support is disabled as default in the settings, so that part of this code is disabled)_, after checking assignments seems like this modification should not affect current implementation. But I would like to revisit this again while we are working on multiple cameras.